vllm/config.py

```diff
@@ -1803,6 +1813,14 @@ def _get_and_verify_max_len(
     return int(max_model_len)
 
 
+def get_min_sliding_window(
+        sliding_window: Union[int, List[Optional[int]]]) -> int:
+    if isinstance(sliding_window, list):
+        return min([s for s in sliding_window if s is not None])
```

Comment:
```suggestion
        return min(s for s in sliding_window if s is not None)
```

No need to iterate twice here.